src/main/scala/com/thoughtworks/sbtApiMappings/ApiMappings.scala

```diff
@@ -45,13 +45,13 @@ object ApiMappings extends AutoPlugin {
       inTask(doc) {
         Seq(
           autoAPIMappings := true,
-          apiMappings ++= {
+          apiMappings := {
             val rules = apiMappingRules.value
             dependencyClasspath.value.collect {
               case jar @ rules.extract(url)
                   if !apiMappings.value.exists(_._1 == jar.data) =>
                 jar.data -> url
-            }(collection.breakOut(Map.canBuildFrom))
+            }(collection.breakOut(Map.canBuildFrom)) ++ apiMappings.value
```

Comment:
Exactly. In this case, the existing value of `apiMappings` is appended to the mappings generated by the plugin, and therefore the existing value takes priority.

Reply:
Could you create a scripted-test to demonstrate the behavior difference?

Reply:
I'm not sure. Generally this is important when multiple independent plugins set the same setting, in which case sbt orders them non-deterministically.

Reply:
I think sbt plugins are loaded in partial order, according to https://www.scala-sbt.org/1.x/docs/Plugins.html#Plugin+dependencies

So we can always add plugin dependency to ensure the order more or less.

Reply:
See:
- https://github.com/sbt/sbt/issues/1875